package.json

```diff
@@ -1,27 +1,35 @@
 {
-  "name": "outline-shadowsocksconfig",
-  "version": "0.0.8",
+  "name": "outline_shadowsocksconfig",
+  "author": "The Outline Team",
+  "version": "0.1.0",
+  "private": true,
   "license": "Apache-2.0",
   "scripts": {
-    "build": "tsc",
-    "test": "yarn build && jasmine --config=jasmine.json",
-    "clean": "rm -rf build/*.spec.* node_modules/"
+    "build": "bazel build //...",
+    "test": "bazel test //...",
+    "clean": "bazel clean --expunge"
```

Comment:
`&& echo "there is a terrible bug if you have to run this, unless you just upgraded some toolchain bit"`